dll/ntdll/def/ntdll.spec

```diff
@@ -1158,6 +1158,7 @@
 @ stub -version=0x600+ RtlSetProcessDebugInformation
 @ cdecl RtlSetProcessIsCritical(long ptr long)
 @ stdcall RtlSetSaclSecurityDescriptor(ptr long ptr long)
+@ stdcall RtlSetSearchPathMode(long)
```

Comment:
This is Vista+

Reply:
@HBelusca  please read the description of JIRA ticket https://jira.reactos.org/browse/CORE-17586 ! Does that answer/invalidate your concern?

Reply:
In my XPSP3 kernel32.dll there is an export for SetSearchPathMode:
![image](https://github.com/reactos/reactos/assets/33393466/d975fb63-564b-49bb-9ec0-83a32ad01b3f)

But in my XPSP3 ntdll.dll there is **no** export for RtlSetSearchPathMode
![image](https://github.com/reactos/reactos/assets/33393466/fd841fb7-bef9-44f7-8f3d-d3d6dce18803)

So we must find a solution here, that adds the one in kernel32.dll, but not the one in ntdll.dll